### ISSUE TITLE
panic in execinquery@v1.2.0/execinquery.go:52

### ISSUE THREAD

**Issue by rski** [lufeee/execinquery#15]
This is when running execinquery as part of golangci-lint:
```
ERRO [runner] Panic: execinquery: package "provider" (isInitialPkg: true, needAnalyzeSource: true): runtime error: invalid memory address or nil pointer dereference: goroutine 201225 [running]:
runtime/debug.Stack()
	runtime/debug/stack.go:24 +0x65
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe.func1()
	github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner_action.go:101 +0x155
panic({0xef29c0, 0x190af10})
	runtime/panic.go:838 +0x207
go/types.(*Package).Path(...)
	go/types/package.go:31
github.com/lufeee/execinquery.linter.run.func1({0x12641a0?, 0xc048398e80})
	github.com/lufeee/execinquery@v1.2.0/execinquery.go:52 +0xbe
golang.org/x/tools/go/ast/inspector.(*Inspector).Preorder(0xc058c5adb0, {0xc00f8994e0?, 0x1?, 0x0?}, 0xc000411cf0)
	golang.org/x/tools@v0.1.11-0.20220316014157-77aa08bb151a/go/ast/inspector/inspector.go:77 +0x9a
github.com/lufeee/execinquery.linter.run({0xc0001eff40?, 0xc0002ac0a0?}, 0xc0657389c0)
	github.com/lufeee/execinquery@v1.2.0/execinquery.go:44 +0xd5
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyze(0xc0129c12e0)
	github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner_action.go:187 +0x9c4
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe.func2()
	github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner_action.go:105 +0x1d
github.com/golangci/golangci-lint/pkg/timeutils.(*Stopwatch).TrackStage(0xc001133ae0, {0x105deea, 0xb}, 0xc00f899748)
	github.com/golangci/golangci-lint/pkg/timeutils/stopwatch.go:111 +0x4a
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe(0xc02f849800?)
	github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner_action.go:104 +0x85
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*loadingPackage).analyze.func2(0xc0129c12e0)
	github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner_loadingpackage.go:80 +0xb4
created by github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*loadingPackage).analyze
	github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner_loadingpackage.go:75 +0x1eb 
WARN [runner] Can't run linter goanalysis_metalinter: goanalysis_metalinter: execinquery: package "provider" (isInitialPkg: true, needAnalyzeSource: true): runtime error: invalid memory address or nil pointer dereference 
ERRO Running error: 1 error occurred:
	* can't run linter goanalysis_metalinter: goanalysis_metalinter: execinquery: package "provider" (isInitialPkg: true, needAnalyzeSource: true): runtime error: invalid memory address or nil pointer dereference
 ```
I'm not sure what caused this in the code checked, I will dig into it when i find some time

**Comment by rski** [comment]
probably same thing as what https://github.com/lufeee/execinquery/pull/13 fixes

**Comment by ldez** [comment]
Fixed by #13
